src/main/kotlin/com/sourcegraph/cody/CodyToolWindowContent.kt

```diff
@@ -52,8 +53,9 @@ class CodyToolWindowContent(private val project: Project) : UpdatableChat {
   private val sendButton: JButton
   private var inProgressChat = CancellationToken()
   private val stopGeneratingButton =
-      JButton("Stop generating", IconUtil.desaturate(AllIcons.Actions.Suspend))
+    JButton("Stop generating", IconUtil.desaturate(AllIcons.Actions.Suspend))
   private val recipesPanel: JBPanelWithEmptyText
+  private val subscriptionPanel: JPanel
```

Comment:
This file is already a mess. If there is a chance to add anything in a separate file in the first place we should go for it.